### DISCUSSION TITLE
EmuDeck Support

### DISCUSSION THREAD

**Discussion by ZaxBit** [retronas/discussions#3] [Ideas]
Hiya! RetroNAS has been awesome and I'm wondering if there's any easy way it could support mapping to the way [EmuDeck for Steam Deck](https://github.com/dragoonDorise/EmuDeck) sets up its folder structure? I know it's been asked around before but there weren't any clear answers.

My current plan was to create some symlinks just like the MiSTER integration, but for [EmuDeck's folder structure](https://github.com/dragoonDorise/EmuDeck/tree/main/roms). That way I could have an easy way to keep things in sync in tandem with [SyncThing on my Deck](https://github.com/dragoonDorise/EmuDeck/wiki/community-creations#how-to-set-up-syncthing). I would love to share my results when I'm done tweaking but am wondering how there might be other ways there could be connectivity? I think my biggest head scratcher will be how EmuDeck handles saves and states wildly different from RetroNAS (by emulator vs. by system).

**Comment by sairuk** [comment]
yes we can look at this

**Reply by sairuk** [reply]
If they are wildly different in the save space, we simply wouldn't support it and would just do basic rom folders, but i can take a look at the repo later today

**Reply by sairuk** [reply]
Basic EmuDeck support is now available in testing

**Reply by ZaxBit** [reply]
thanks @sairuk I'll check it out!!

**Comment by coldjim** [comment]
@ZaxBit if you ever get anywhere with this, i'd be grateful if you could share. Sounds like a greet idea

**Reply by ZaxBit** [reply]
Will do. I think I got close but am a little out of touch with the way SyncThing works-- I was able to begin syncing all the emulation save folders but needed to match the same on my Desktop PC as well. SyncThing would be a lot easier to use if it allowed for easy selective syncing, but instead you need to structure everything folder by folder (unless you get very intricate with the ignore list).
